Example/ChatWallet/ApplicationLayer/Configurator/AppearanceConfigurator.swift

```diff
@@ -0,0 +1,19 @@
+import UIKit
```

Comment:
all the files are in Example/chatWallet dir. Chat app is not a wallet. I think Example/Chat would be good